contracts/StakeHub.sol

```diff
@@ -256,6 +274,7 @@ contract StakeHub is SystemV2, Initializable, Protectable {
         downtimeJailTime = 2 days;
         felonyJailTime = 30 days;
         maxFelonyBetweenBreatheBlock = 2;
+        maxNodeIDs = 5;
```

Comment:
uint256 public constant INIT_MAX_NUMBER_NODE_ID = 5;
    bool maxNodeIDsInitialized

    function maxNodeIDsInitializer() internal{
        if (!maxNodeIDsInitialized)
        	maxNodeIDs = INIT_MAX_NUMBER_NODE_ID
        	maxNodeIDsInitialized = true
        }
    }
    
    then use `maxNodeIDsInitializer` for new added functions

Reply:
`initialize` will not be called when maxwell hardfork enabled.

Reply:
How about this, add a getter method, and return the default when empty.

```solidity
function getMaxNodeIDs() external view returns (uint256) {
        if (maxNodeIDs == 0) {
            return 5;
        }
        return turnLength;
    }

```